src/views/Region.vue

```diff
@@ -256,10 +260,16 @@ export default {
     </div>
   </div>
   <div v-else class="container-fluid">
-    <search-box></search-box>
-
+    <b-navbar class="py-0" type="light" variant="light" fixed="top">
+      <b-button class="btn btn-sm btn-secondary mr-3" style="pointer-events: none;">Go to:</b-button>
```

Comment:
I might suggest styling this section as text and links- in particular, the "Go To" button looks clickable, but isn't. (I had to go read the source code to be sure)

We've avoided UX comments so far, but the general UX guideline is that ["a link takes you to other place, a button does something."](https://ux.stackexchange.com/a/5494). When "navigation cues" are used purely as style (like Go to), it may leave your users confused.

Bootstrap allows `btn-link` style, so you can preserve all of the markup you're already using.

Reply:
Replaced the Go To button and added `btn-link` to the link buttons in a024633